lib/models/patient_source.js

```diff
@@ -6,38 +6,41 @@ module.exports = class MongoDBPatientSource {
   /* Insert documentation here
     */
 
-  constructor(connectionInfo) {
-    mongoose.connect(connectionInfo);
+  constructor(connectionInfo = null) {
```

Comment:
How will the "Patient Source", "Measure Source", and "ValueSet Source" work with running in the client? I am thinking there needs to be an optional parameter so that the Bonnie team can pass in data we want to use during execution. Thoughts?